web/src/main/java/se/inera/intyg/intygstjanst/web/service/dto/CertificateExportPageDTO.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright (C) 2022 Inera AB (http://www.inera.se)
+ *
+ * This file is part of sklintyg (https://github.com/sklintyg).
+ *
+ * sklintyg is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * sklintyg is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package se.inera.intyg.intygstjanst.web.service.dto;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.springframework.data.domain.Page;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+public class CertificateExportPageDTO {
+
+    private long total;
+    private long totalRevoked;
+    private Page<CertificateXmlDTO> certificateXmlPage;
```

Comment:
Page includes extra attributes that we don't have a need for in the api. The api would be simpler if you only returned a List<CertificateXmlDTO>.